### PR TITLE
Empty Gallery Crash Fix

### PR DESCRIPTION
**Pull request by joewalsh**

### CODE REVIEW

**Comment by coreyfloyd** [comment]
👍

**Comment by montehurd** [comment]
@joewalsh what article were u seeing this on?

**Comment by joewalsh** [comment]
@montehurd 

https://en.wikipedia.org/wiki/Bishop_McDevitt_High_School_(Wyncote,_Pennsylvania)
https://en.wikipedia.org/wiki/Ancillae_Assumpta_Academy

related issues:

File: Wikipedia/Code/WMFArticleViewController.m [hunk 1]
```diff
@@ -1015,7 +1017,9 @@ - (void)imageViewDidTap:(UITapGestureRecognizer*)tap {
 
     WMFArticleImageGalleryViewController* fullscreenGallery = [[WMFArticleImageGalleryViewController alloc] initWithArticle:self.article];
     fullscreenGallery.referenceViewDelegate = self;
-    [self presentViewController:fullscreenGallery animated:YES completion:nil];
+    if (fullscreenGallery != nil) {
+        [self presentViewController:fullscreenGallery animated:YES completion:nil];
```
Comment (montehurd) [inline Wikipedia/Code/WMFArticleViewController.m]: Do we maybe want a "wmf_safePresentViewController"?

**Comment by montehurd** [comment]
👍

**Pull request merged**